pkg/certificate/secret.go

```diff
@@ -110,6 +111,7 @@ func (m *Manager) applySecret(secretKey types.NamespacedName, secretType corev1.
 						Name:        secretKey.Name,
 						Namespace:   secretKey.Namespace,
 						Annotations: map[string]string{},
+						Labels:      names.RelationshipLabels,
```

Comment:
@qinqon 
since you asked on https://github.com/nmstate/kubernetes-nmstate/pull/770
to remove the global var and just always calculate it

do we want to do here the same ?
always read the env vars?
or we want to change the constructor to have another parameter,
but then we will need to save that parameter in some global var

Reply:
Don't use a global pass it over the functions until it reaches this, is better to be explicit with dependencies.

Reply:
better to just run the function isn't it ?
won't it be too much sending in the call stack ?

Reply:
ah its will be part of Manager, and the function is a method of manager so we are good

Reply:
Just note that we will need to add logic of https://github.com/nmstate/kubernetes-nmstate/pull/770
also to kubemacpool because kubemacpool will also need to pass the argument

Reply:
Done, just need to think on the tests if any